monkey/agent_plugins/credentials_collectors/chrome/src/plugin.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import Any, Mapping, Sequence
+
+from common.credentials import Credentials
+from common.event_queue import IAgentEventPublisher
+from common.types import AgentID, Event
+from infection_monkey.i_puppet import TargetHost
+
+from .chrome_credentials_collector_builder import build_chrome_credentials_collector
+
+logger = logging.getLogger(__name__)
+
+
+class Plugin:
+    def __init__(self, *, agent_id: AgentID, agent_event_publisher: IAgentEventPublisher, **kwargs):
+        self._agent_id = agent_id
+        self._agent_event_publisher = agent_event_publisher
+
+    def run(
+        self, *, host: TargetHost, options: Mapping[str, Any], interrupt: Event, **kwargs
```

Comment:
I'm not seeing options getting parsed. Do we have options for this plugin?

Reply:
ATM I don't believe there are any options.